subprojects/packagefiles/libxml2/meson.build

```diff
@@ -78,9 +78,23 @@ if math_lib.found()
 endif
 
 if meson.version().version_compare('>= 0.60')
-  iconv_lib = dependency('iconv', required: false)
+  iconv_lib = dependency('iconv', required: get_option('iconv'))
+elif get_option('iconv').disabled()
+  iconv_lib = disabler()
 else
-  iconv_lib = dependency('', required: false)
+  iconv_lib = dependency('iconv', required: false)
```

Comment:
This is probably a waste of time, since iconv won't ever exist as a dependency without the builtin check. Upstream iconv certainly doesn't ship a pkg-config file.